R/ocean_entry_success.R

```diff
@@ -1,39 +1,55 @@
-                                          `intercept 16` = 1.2, `intercept 17` = 1.2, `intercept 18` = -3.5,
-                                          `intercept 19` = -3.5, `intercept 20` = -3.5, `intercept 21` = 1.2,
-                                          `intercept 22` = 1.2, `intercept 23` = -1.308341, `intercept 24` = 1.2,
-                                          `intercept 25` = -1.9841364, `intercept 26` = -1.9841364, `intercept 27` = 2.5000007,
-                                          `intercept 28` = -3.5, `intercept 29` = -3, `intercept 30` = -0.9,
-                                          `intercept 31` = 1.2, months = 0.35)){
+                                ..ocean_entry_success_int = c(
+                                  `intercept Upper Sacramento River` = -0.5108849,
+                                  `intercept Antelope Creek` = 1.2,
+                                  `intercept Battle Creek` = 1.2,
+                                  `intercept Bear Creek` = 1.2,
+                                  `intercept Big Chico Creek` = 1.2,
+                                  `intercept Butte Creek` = -3.3233638,
+                                  `intercept Clear Creek` = 1.2,
+                                  `intercept Cottonwood Creek` = 1.2,
+                                  `intercept Cow Creek` = 1.2,
+                                  `intercept Deer Creek` = -3.2304288,
+                                  `intercept Elder Creek` = 1.2,
+                                  `intercept Mill Creek` = -3.4148335,
+                                  `intercept Paynes Creek` = 1.2,
+                                  `intercept Stony Creek` = 1.2,
+                                  `intercept Thomes Creek` = 1.2,
+                                  `intercept Upper-mid Sacramento River` = 1.2,
+                                  `intercept Sutter Bypass` = 1.2,
+                                  `intercept Bear River` = -3.5,
+                                  `intercept Feather River` = -3.5,
+                                  `intercept Yuba River` = -3.5,
+                                  `intercept Lower-mid Sacramento River` = 1.2,
+                                  `intercept Yolo Bypass` = 1.2,
+                                  `intercept American River` = -1.308341,
+                                  `intercept Lower Sacramento River` = 1.2,
+                                  `intercept Calaveras River` = -1.9841364,
+                                  `intercept Cosumnes River` = -1.9841364,
+                                  `intercept Mokelumne River` = 2.5000007,
+                                  `intercept Merced River` = -3.5,
+                                  `intercept Stanislaus River` = -3,
+                                  `intercept Tuolumne River` = -0.9,
+                                  `intercept San Joaquin River` = 1.2),
+                                .months = 0.35){
```

Comment:
not sure if we need to prefix all these with "intercept", I'm guessing it would be nice to be able to index the `..ocean_entry_success_int` with a watershed name `..ocean_entry_success_int["Upper Sacramento River"]` ..thoughts?

Reply:
I agree. I will fix that now.